addons/l10n_ch/models/account_invoice.py

```diff
@@ -79,16 +77,20 @@ def _space_isr_number(isr_number):
             return res
 
         for record in self:
-            if record.name and record.invoice_partner_bank_id and record.invoice_partner_bank_id.l10n_ch_postal:
-                invoice_issuer_ref = re.sub('^0*', '', record.invoice_partner_bank_id.l10n_ch_postal)
+            isr_number = False
+            isr_number_spaced = False
+            if record.name and record.l10n_ch_isr_subscription:
+                invoice_issuer_ref = re.sub('^0*', '', record.l10n_ch_isr_subscription)
```

Comment:
Not sure what this is for

Reply:
This is just a copy of sub `_format_isr_subscription`.

This is to remove the starting `0`. Like to convert 010000234 to 1-23-4